sphinx_gallery/gen_rst.py

```diff
@@ -872,8 +878,15 @@ def rst_blocks(script_blocks, output_blocks, file_conf, gallery_conf):
             code_rst = codestr2rst(bcontent, lang=gallery_conf['lang'],
                                    lineno=lineno) + '\n'
             if is_example_notebook_like:
-                example_rst += code_rst
-                example_rst += code_output
+                if gallery_conf['thebelab']:
+                    example_rst += ".. container:: thebelab\n"
```

Comment:
Here instead of adding all the containers and such, perhaps we can just configure `sphinx-thebelab` with HTML selectors that are unique to sphinx-gallery

Reply:
So you can get multiple outputs, so I think there is a need to have a container to group these together in order to get it work properly.